mantaray/views.py

```diff
@@ -542,6 +547,27 @@ def on_part(self, nick: str, reason: str | None) -> None:
             show_in_gui=self.server_view.should_show_join_leave_message(nick),
         )
 
+    def on_kick(self, channel: str, kicker: str, kicked_nick: str, reason: str | None) -> None:
+        self.userlist.remove_user(kicked_nick)
+        if reason is None:
+            reason = ''
+        if kicker == self.server_view.core.nick:
+           kicker_tag = 'self-nick'
+        else:
+           kicker_tag = 'other-nick'
+        kicked_tag=[]
+        if kicked_nick == self.server_view.core.nick:
```

Comment:
In this case, it might be to display a completely different message, e.g. "Akuli kicked you from #foo (Reason: fix your internet)". This message should probably be in red (with everything tagged as `"error"`), so that you see it very clearly.

Reply:
Even better, it would be good if the kick message contained instructions for rejoining to the channel, e.g. `Type '/join #foo' if you dare`.

Reply:
tag added.